packages/integration-tests/src/components/delegates-focus-tab-navigation/test-focusable-coverage/focusable-coverage.spec.js

```diff
@@ -9,16 +9,18 @@ const assert = require('assert');
 const URL = '/focusable-coverage';
 
 describe('sequential focus navigation coverage', () => {
+    before(() => {
+        // This test suite fails in ie11 for some unknown reason
+        if (browser.capabilities.commonName === 'ie11') {
```

Comment:
With wdio 6, `browser` is not available in the `describe` block anymore. The filtering can be done in a `before` block.